doc/developer_guide/developer_guide.md

```diff
@@ -15,17 +15,17 @@ npm run lint:ci // No autofix enabled
 ### Unit tests
 
 ```bash
- npm run test // Runs both test in parallel
- npm run test:dom
- npm run test:node
+npm run test // Runs both test in parallel
```

Comment:
```suggestion
npm run test # Runs both test in parallel
```

// is not a comment in bash, which would prevent the coping of this code block

Reply:
Smart!